src/components/HoverInlineText/index.tsx

```diff
@@ -0,0 +1,67 @@
+import Tooltip from 'components/Tooltip'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+const TextWrapper = styled.div<{ margin: boolean; link: boolean; fontSize?: string; adjustSize?: boolean }>`
+  position: relative;
+  margin-left: ${({ margin }) => margin && '4px'};
+  color: ${({ theme, link }) => (link ? theme.blue1 : theme.text1)};
+  font-size: ${({ fontSize }) => fontSize ?? 'inherit'};
+
+  :hover {
+    cursor: pointer;
+  }
+
+  @media screen and (max-width: 600px) {
+    font-size: ${({ adjustSize }) => adjustSize && '12px'};
+  }
+`
+
+const HoverInlineText = ({
+  text,
+  maxCharacters = 20,
+  margin = false,
+  adjustSize = false,
+  fontSize,
+  link,
+  ...rest
+}: {
+  text: string
+  maxCharacters?: number
+  margin?: boolean
+  adjustSize?: boolean
+  fontSize?: string
+  link?: boolean
+}) => {
+  const [showHover, setShowHover] = useState(false)
+
+  if (!text) {
+    return <span></span>
+  }
+
+  if (text.length > maxCharacters) {
```

Comment:
I wonder if you could simplify this block with css ch units, e.g. max-width: ${maxCharacter}ch

Reply:
Oh yeah - how does this handle the overflow display? @justindomingue

Reply:
max-width in ch units + text-overflow: ellipsis I was thinking. Would have to play with it for the specific